examples/snippets/modules/ROOT/examples/sensor-data-scala/.gitignore

```diff
@@ -0,0 +1,30 @@
+etc/bootstrap-local.conf
```

Comment:
Why is the entire sensor-data-scala copied in here @skyluc ? Can we avoid it?

Reply:
As discussed offline, the idea is that eventually we get rid of the standalone sensor-data-xxx examples in favor of these snippets. The sensor data examples were always meant as getting started info more than full examples.